src/01-Learning-JS/08-load-data.md

```diff
@@ -234,12 +240,17 @@ In this case, load the following CSV file: `nc_absentee_mail_2024.csv`.
 2. In the second codeblock, use `console.log()` to log the following properties to the console: `.name` and `.size`.
 3. Still in the second codeblock, add a second `console.log()` and log the first object in the Array to the console.
 
-```javascript
-// Your FileAttachment() code goes here
+```js
+const absenteeMail = FileAttachment("./../data/nc-voters/nc_absentee_mail_2024.csv").csv({typed: true})
 ```
 
-```javascript
-// Your other code with your variable goes here
+```js
+// This code returned undefined, undefined: console.log(absenteeMail.size,absenteeMail.name)
+
+console.log(absenteeMail.length)
+//This code worked. Is it something to do with observable?
+
+console.log(absenteeMail[0])
```

Comment:
My bad about the .name and .size properties of FileAttachment(). They're only accessible before you type the data, so once we apply something like the encoder of .csv(), in this case, we can't access those properties anymore.

I'm going to revise this assignment a smidge to provide clearer guidance here.

But, FTR, this looks great.